src/reducers/api.spec.tsx

```diff
@@ -0,0 +1,21 @@
+import reducer, { actions, initialState } from './api';
+
+describe(__filename, () => {
+  describe('reducer', () => {
+    it('initializes the state', () => {
+      const state = reducer(undefined, { type: 'SOME_ACTION' } as any);
```

Comment:
Why not just `reducer(undefined, {})`, or does TS get mad about that?

Reply:
I believe so and that would make sense given that a redux action should have at least a `type`.